Penteract/Assets/Scripts/GameController.h

```diff
@@ -47,11 +47,11 @@ class GameController : public Script {
 	float focusDistance = 100.f;
 	float transitionSpeed = 1000.f;
 
-	int obtainedPowerUps = 0; // PowerUp counter
+	static inline bool isGameplayBlocked = false;		// isGameplayBlocked is used to stop all the gameplay without pausing the game. In Fangand Onimaru.cpp - PlayAnimation(), it forces toe player to stay in an idle state. In Player.cpp - Update(), controls that the player becomes static and doesn't move.
```

Comment:
Since this is a quite important variable, this should be a private variable that is accessed by a getter and setter.

Reply:
I would like @Jordi-Gil to confirm this, since that was the solution we agreed to use